integration_tests/src/main/python/delta_lake_utils.py

```diff
@@ -198,3 +199,10 @@ def setup_delta_dest_tables(spark, data_path, dest_table_func, use_cdf, enable_d
     for name in ["CPU", "GPU"]:
         path = "{}/{}".format(data_path, name)
         setup_delta_dest_table(spark, path, dest_table_func, use_cdf, partition_columns, enable_deletion_vectors)
+
+def enable_in_commit_ts():
```

Comment:
This function returns a list of supported values for the `enableInCommitTimestamp` config based on the spark version. Because its behavior is tightly coupled with how the caller should use the returned value, I'm not sure if this a util function to place in a shared file. How about just moving this function to `delta_lake_time_travel_test.py`?